tests/objects/tobjpragma.nim

```diff
@@ -8,7 +8,7 @@ discard """
 2
 3
 '''
-disabled: "true"
+    disabled: "true"
```

Comment:
Disabled around 2015-03 for CI sake
There seems to be a ton of issues referencing this:
https://github.com/nim-lang/Nim/issues/15752
https://github.com/nim-lang/Nim/issues/9668
https://github.com/nim-lang/Nim/issues/10082
https://github.com/nim-lang/Nim/pull/10227
https://github.com/nim-lang/Nim/issues/4763

seems to have been fixed on https://github.com/nim-lang/Nim/pull/10227

working on my pc

Reply:
Enable and let's see what happens.